badges/README.md

```diff
@@ -5,20 +5,24 @@
 ```
 {
   'id': string,
-  'title': string,
+  'name': string,
   'description': string,
-  'imgUrl': string,
-  'users': [
-    rdsUserId,
-  ],
+  'imageUrl': string,
+  'createdBy': string,
+  'createdAt': { 'date': string, 'time': string }
 }
 ```
 
 ## **Requests**
 
-|              Route               |           Description            |
-|:--------------------------------:|:--------------------------------:|
-|    [GET /badges](#get-badges)    | Returns all badges in the system |
+|              Route                                                      |           Description                   |
+|:-----------------------------------------------------------------------:|:---------------------------------------:|
+|    [GET /badges](#get-badges)                                           |   Returns all badges in the system      |
+|    [GET /badges/:username](#get-badgesusername)                         |   Returns badges assigned to the user   |
+|    [POST /badges](#post-badges)                                         |   Creates a new badge                   |
+|    [POST /badges/assign/:username](#post-badgesassignusername)          |   Assigns a badge to the user           |
+|    [DELETE /badges/unassign/:username](#delete-badgesunassignusername)  |   Unassigns a badge from user badges    |
```

Comment:
badges/assign
body : {
user, badges[]
}

Reply:
Description: [Assign and Unassign Badges](7f36e92ed10393af32e1bfc28f6bfe069493cb39)